gef-ui/server/controllers/about-facility/index.test.js

```diff
@@ -671,12 +671,9 @@ describe('controllers/about-facility', () => {
       mockRequest.body.shouldCoverStartOnSubmission = 'true';
       mockRequest.body.hasBeenIssued = 'true';
       mockRequest.body.monthsOfCover = '10';
-      mockRequest.body['cover-start-date-day'] = '01';
```

Comment:
out of curiosity does cover-start-date get overwritten if we have `shouldCoverStartOnSubmission = true`?

Reply:
Yes, by `portal-api` when it receives the PUT request from `gef-ui` (you can see this in the constructor in `portal-api\src\v1\gef\models\facilities.js`)